README.md

```diff
@@ -90,6 +90,39 @@ Runtime Environment (node v10.15.1 on linux)
 CLI version 3.0.1
 ```
 
+## Install locally via yarn (alternative install)
+
+required: current version of yarn installed (should also work with npm)
+
+create a file `package.json`within your project with the following content
+
+```javascript
+{
+  "name": "example",
+  "version": "1.0.0",
+  "dependencies": {
+    "asciidoctor": "^2.0.3",
+    "asciidoctor-pdf": "^1.0.0-alpha.3"
+  },
+  "scripts": {
+    "render-pdf": "asciidoctor-pdf <path to your adoc-file>"
+  }
+}
+```
+
+the commands
+
+```javascript
```

Comment:
This is no JavaScript, actually a shell script

`````suggestion
```
`````

Reply:
argh. copy and paste...